### PR TITLE
Add as_edges_source/as_edges_target to NodePopulations

### PR DESCRIPTION
**Pull request by tomdele**

### CODE REVIEW

**Comment by tomdele** [comment]
I am not so sure about the naming and description... 
@asanin-epfl  do you think this is clear enough ?

File: bluepysnap/nodes.py [hunk 1]
```diff
@@ -155,6 +155,16 @@ def _property_names(self):
     def _dynamics_params_names(self):
         return set(utils.add_dynamic_prefix(self._population.dynamics_attribute_names))
 
+    def as_edges_source(self):
+        """Set of edges population that use this node population as source."""
```
Comment (asanin-epfl) [inline bluepysnap/nodes.py]: Missing plural **s** """Set of edges population**s**...

Comment (tomdele) [reply bluepysnap/nodes.py]: thanks ! I changed it in the last commit.

**Comment by tomdele** [comment]
Yes you are perfectly right about the naming. 
I feel like `source_edges` is not super clear. What about `source_in_edges` ?

**Comment by asanin-epfl** [comment]
> What about `source_in_edges`?

Works good too. The prefix `as` only worried me.

**Comment by tomdele** [comment]
perfect thanks for the feedback I will change this

**Pull request merged**